docs/conf.py

```diff
@@ -9,12 +9,12 @@
 
 # -- Project information -----------------------------------------------------
 
-project = 'supersid'
+project = 'SuperSIDpy'
```

Comment:
sidpy rather than supersidpy?

Reply:
Just fixed there thank you for pointing this out :)